app/views/directives/lifecycle-hook.html

```diff
@@ -1,133 +1,56 @@
-        </div>
-
-        <div class="form-group">
-          <label class="required">Command</label>
-          <edit-command args="hookParams.execNewPod.command" is-required="true"></edit-command>
-        </div>
-
-        <div class="form-group">
-          <label>Environment Variables</label>
-          <key-value-editor
-            entries="hookParams.execNewPod.env"
-            key-validator="[a-zA-Z_][a-zA-Z0-9_]*"
-            key-validator-error-tooltip="A valid environment variable name is an alphanumeric (a-z and 0-9) string beginning with a letter that may contain underscores."
-            add-row-link="Add environment variable"></key-value-editor>
-          <div class="help-block">
-            Environment variables to supply to the hook pod's container.
-          </div>
-        </div>
-
-        <div class="form-group">
-        <label>Volumes</label>
-        <ui-select multiple placeholder="Select volume" ng-model="hookParams.execNewPod.volumes" ng-disabled="view.isDisabled">
-          <ui-select-match>{{$item}}</ui-select-match>
-          <ui-select-choices repeat="volume in availableVolumes | filter : $select.search">
-            <div ng-bind-html="volume | highlight : $select.search"></div>
-          </ui-select-choices>
-        </ui-select>
-        <div class="help-block">
-          List of named volumes to copy to the hook pod.
-        </div>
-        </div>
-      </div>
-
-      <div ng-if="action.type === 'tagImages'">
-        <div ng-repeat="tagImage in hookParams.tagImages">
-          <div ng-if="hookParams.tagImages.length === 1">
-            <div class="form-group">
-              <label class="required">Container Name</label>
-              <ui-select ng-model="tagImage.containerName" ng-disabled="view.isDisabled" required>
-                <ui-select-match>{{$select.selected}}</ui-select-match>
-                <ui-select-choices repeat="container in (availableContainers | filter : $select.search)">
-                  <div ng-bind-html="container | highlight : $select.search"></div>
-                </ui-select-choices>
-              </ui-select>
-              <div class="help-block">
-                Use the image for this container as the source of the tag.
-              </div>
-            </div>
-            <div class="form-group">
-              <label class="required">Tag As</label>
-              <istag-select model="istagHook" allow-custom-tag="true" select-disabled="view.isDisabled"></istag-select>
+<h4>
+  {{type}} Hook
+  <small class="text-muted">{{hookParams.execNewPod ? "Runs a command in a new pod based on the specified container" : "Tags image on deployment success"}}</small>
+  <span ng-switch="hookParams.failurePolicy">
+    <small ng-switch-when="Ignore">(Continue with deployment on failure)</small>
+    <small ng-switch-when="Abort">(Abort deployment on failure)</small>
+    <small ng-switch-when="Retry">(Continue with deployment on failure)</small>
+  </span>
```

Comment:
I don't think we should style this as help text since it's actual data in the hook and is not always the same value. I'd say we call it out specifically as failure policy below the header.